events/git-and-github.md

```diff
@@ -129,11 +138,30 @@ Share the above via a facebook post along with a thanks for the audience.
 - Skip hands-on sessions. Hands on sessions are the main reason we conduct workshops, else audience would have simply watched a tutorial video. Repeat demos if required. Provide individual assistance. Make sure audience get their hands dirty.
 - Skip questions. Git contains many non-intuitive ideas. Audience may be confused while you are explaining the concepts. Re-explain things in a different way if necessary. Encourage audience to speak out if they have confuse. Tell them that Git is non-intuitive and its okay to ask small doubts.
 
-
+## KOSS Intro
+This git and github workshop will have a significant part, introducing them about KOSS (4-5 slides worth it material). Focusing on when we do selections, why we do it at that time and what we expect from them - no need to introduce to open source.
 ## Miscellaneous
 - We can also consider to conduct a Advanced Git workshop, where concepts like `rebase`, `stash` and `bisect` are taught. The duration of this workshop can be shorter than duration of a regular workshop(3 hours). If such a thing is being planned, make sure to give some git excerises in the gap between git basic workshop and git advanced workshop, so that they are in touch with git concepts.
 - `switch` is the new preferred way to switch branches. This might be included in the upcoming workshops. Refer [this](https://www.banterly.net/2021/07/31/new-in-git-switch-and-restore/).
 
+## Feedback Form
+Include a feedback form at the end of the slides and ask the audience to share their experience and suggestions.
+Some questions to include in the feedback form can be:  
+- Can you now explain Git-Github to someone totally new in this field?
+- Can you revert a commit with errors now?
+- I don't want the changes made by you recently, which are not yet committed. Can you quickly roll them back?
+- How well did you grasp the concept of Git branching?
+- Are you condifent enough to push that important file into the github repository? Or try pulling the doc?
+- How well did you understand the flow of making a Pull Request?
+- Were the workshop materials, such as slides and handouts, clear and helpful in your learning process?
+- Do you feel more confident in collaborating with others on GitHub after attending this workshop?
+- Were there any specific topics or concepts related to Git and GitHub that you expected to be covered in the workshop but were not?
+- Did you encounter any challenges or areas of confusion during the workshop? If so, please describe them.
+- Did you find the pace of the workshop appropriate, too fast, or too slow?
+- How likely are you to apply the knowledge and skills gained from this workshop in your future projects or work?
+- What more could we have done to improve your learning experience?
+- How relevant and useful did you find the workshop?
+
```

Comment:
Also, mention that including all of the above questions is not important. Try to keep the form short. Also, add a "Any other suggestions?" question at the end.

Some objective questions (1-10 scale) can also be mentioned.